great_expectations/experimental/datasources/postgres_datasource.py

```diff
@@ -3,6 +3,8 @@
 import copy
 import dataclasses
 import itertools
+from collections import namedtuple
```

Comment:
Can you use the `typing.NameTuple`?
https://docs.python.org/3/library/typing.html#typing.NamedTuple